### PR TITLE
Adoption fixes

### PR DESCRIPTION
**Pull request by bdryanovski**
A lot of fixes related to the adoption guide code snippets

* let both left and right sides of the snippets match as much as possible and keep the focus on arguments and code structure
* fixing broken snippets that were not working inside Stackblitz
* remove unneeded code

### CODE REVIEW

File: src/app/migrations/datalist/datalist.2.core.txt
Comment (bbogdanov) [inline src/app/migrations/datalist/datalist.2.core.txt:9]: What is the intention behind this change?

File: src/app/migrations/range/range.1.core.txt
Comment (bbogdanov) [inline src/app/migrations/range/range.1.core.txt:1]: Range and Select are not merged yet. Do you plan to merge this PR after the `range` and `select` are merged?

File: src/app/migrations/toggle/toggle.1.angular.txt
Comment (bbogdanov) [inline src/app/migrations/toggle/toggle.1.angular.txt:9]: What is the intention behind this change?

File: src/app/migrations/toggle/toggle.2.angular.txt
Comment (bbogdanov) [inline src/app/migrations/toggle/toggle.2.angular.txt:12]: Is this necessary? Having in mind that both sides of the example are looking the same

File: src/app/migrations/toggle/toggle.2.core.txt
Comment (bbogdanov) [inline src/app/migrations/toggle/toggle.2.core.txt:10]: Same as the one above.

File: src/app/pages/badge.page.ts
Comment (bbogdanov) [inline src/app/pages/badge.page.ts:18]: Thank you for adding these!

**Review by bbogdanov — CHANGES_REQUESTED** [review]

File: src/app/migrations/datalist/datalist.2.core.txt
Comment (bdryanovski) [reply src/app/migrations/datalist/datalist.2.core.txt:9]: Simplification and most of all working inside stackblitz. Also try to keep both side equal so the focus will be on the code flow and attributes

File: src/app/migrations/range/range.1.core.txt
Comment (bdryanovski) [reply src/app/migrations/range/range.1.core.txt:1]: You are right they slipped (bad fork) - will merge this PR after I merge the other one.

File: src/app/migrations/toggle/toggle.1.angular.txt
Comment (bdryanovski) [reply src/app/migrations/toggle/toggle.1.angular.txt:9]: The left and right sides are not equal .. and there is no reason for the order of the HTML elements but could make someone believe there is.

`ClrToggleWrapper` will find and pick the elements inside its internal template so again there is no reason for reordering but this way both sides look identical and also try to keep the semantic of the HTML clear

File: src/app/migrations/toggle/toggle.2.angular.txt
Comment (bdryanovski) [reply src/app/migrations/toggle/toggle.2.angular.txt:12]: same as above

File: src/app/migrations/toggle/toggle.2.core.txt
Comment (bdryanovski) [reply src/app/migrations/toggle/toggle.2.core.txt:10]: same as above, above

File: src/app/pages/badge.page.ts
Comment (bdryanovski) [reply src/app/pages/badge.page.ts:18]: For nothing

**Review by bbogdanov — APPROVED** [review]

**Review by mathisscott — APPROVED** [review]
Assuming @bbogdanov 's concerns are addressed... 👍

**Pull request merged**
